src/components/blog/post/header/cover.module.scss

```diff
@@ -0,0 +1,5 @@
+.root {
+  background-position: center bottom;
+  background-size: cover;
+  filter: grayscale(100%);
```

Comment:
You can grayscale images using sharp and save some processing power for the clients here

Reply:
:thinking: nice catch, haven't thought of that. I'll look into it.

Reply:
Not feasible on the Aurora website as they have filters and mix blend shenanigans, but if you are just grayscaling it, you can pre-process it. `filters` always take a bunch of power

Reply:
Don't forget to remove it here to test ;p

Reply:
Added your suggestion of using sharp's `grayscale` in 6fd01f6. However I can't take this rule out yet as all cover images right now are external URLs, and are not being processed through sharp. If we get to move all images to the repository we can then remove this, but not until then.